src/SfxWeb/src/app/views/cluster/essentials/essentials.component.ts

```diff
@@ -41,12 +43,15 @@ export class EssentialsComponent extends BaseControllerDirective {
   replicasDashboard: IDashboardViewModel;
   upgradesDashboard: IDashboardViewModel;
   upgradeAppsCount = 0;
+  fmQuorumLossStatus : string;
```

Comment:
change this to a boolean

Reply:
<img width="539" alt="image" src="https://github.com/microsoft/service-fabric-explorer/assets/54608310/ece0a790-5cc3-4619-80d0-c17594af5477">
The data I get from API call is in string type. I can change the variable name to "fmPartitionStatus" if it's causing confusion.